docker/ubuntu_22.04-dev.dockerfile

```diff
@@ -65,30 +74,52 @@ RUN conda uninstall -y conda-libmamba-solver
 RUN conda config --set solver classic
 RUN conda update -y --all && \
     mamba install -y \
-                cmake \
+                make \
+                gxx_linux-64 \
+                gcc_linux-64 \
                 git \
+                cmake \
+                gfortran \ 
                 libblas \
                 liblapack \
+                eigen \
                 hdf5 \
-                setuptools \
+                numpy==1.23 \
```

Comment:
I think we can go with the latest version of numpy as #1485 fixed that issue

Reply:
Let me know when this had been updated @bquan0 and I think we're ready to merge